### DISCUSSION TITLE
Changes to my SearXNG instance not showing up on Website

### DISCUSSION THREAD

**Discussion by masonc-dev** [searxng/searxng#2783] [Q&A]
Hi, I run SearXNG with Nginx and I recently changed the logo and settings file, but It does not update on my live website, I have restart SearXNG, I have restarted my server and I have cleared my cache and I don´t know why its not updating.

NO, Im not using docker.

**Comment by return42** [comment]
Are you using the docker image / if so, I would move this discussion to https://github.com/searxng/searxng-docker/discussions

**Reply by masonc-dev** [reply]
No, I am not using the docker image, I am using SearXNG with Nginx (not docker)
